### DISCUSSION TITLE
How to get a format which can be compatible with <input type="datetime-local">

### DISCUSSION THREAD

**Discussion by tamuhey** [moment/luxon#1136] [Q&A]
How to get a format which can be compatible with `<input type="datetime-local">`?

**Comment by ziishaned** [comment]
Try with following:

```js
DateTime.fromISO(new Date().toISOString()).toISO().slice(0, 16)
```

**Comment by icambron** [comment]
Or:

```js
someDateTime.startOf("minute").toISO({ includeOffset: false, suppressSeconds: true, suppressMilliseconds: true})
```

**Reply by Slembit** [reply]
Stumbled across the same issue, I Used the toFormat function that allows adding extra text using single quotes:
So I added a 'T' between date and time
.toFormat("yyyy-MM-dd'T'HH:mm");
